### DISCUSSION TITLE
Black background icon for macOS?

### DISCUSSION THREAD

**Discussion by dmoncada** [VSCodium/vscodium#625] [Q&A]
I was wondering if someone knew where can I find the .icns for the icon with the black background that @cristianovitorino created for [this](https://github.com/VSCodium/vscodium/issues/313#issue-544825584) issue (I'm referring to the left-hand-side icon on option num. 2 in the poll [here](https://github.com/VSCodium/vscodium/issues/313#issuecomment-635612020))? I navigated to commit [46601d6f](https://github.com/VSCodium/vscodium/commit/46601d6f641127762f2ce754883ac2481e170652#diff-4fe51c2eae4f547945188d11577ca7aeda422b4ab3fe3ec3963329e80cc85eb3) but I could only find the .icns for the icon with the light background =/

**Comment by setanarut** [comment]
@dmoncada https://yadi.sk/d/4bLeqKEO8bgOsg
![codium_app](https://user-images.githubusercontent.com/36481442/106512042-26b95a00-64e2-11eb-9cb3-f4001202411b.png)
